base/shell/explorer/util.cpp

```diff
@@ -237,3 +237,24 @@ GetVersionInfoString(IN LPCWSTR szFileName,
 
     return bRet;
 }
+
+HRESULT ShellExecuteCommand(PCWSTR Command)
+{
+    WCHAR szCmd[MAX_PATH * 2];
+    int len = PathProcessCommand(Command, szCmd, _countof(szCmd), PPCF_ADDARGUMENTS | PPCF_FORCEQUALIFY);
+    if (len <= 0) // Could not resolve the command, just use the input
+    {
+        HRESULT hr = StringCchCopyW(szCmd, _countof(szCmd), Command);
+        if (FAILED(hr))
+            return hr;
+    }
+    PWSTR pszArgs = PathGetArgsW(szCmd);
+    if (*pszArgs)
+        pszArgs[-1] = UNICODE_NULL; // Split szCmd program and arguments
+    PathUnquoteSpacesW(szCmd);
+
+    SHELLEXECUTEINFOW sei = { sizeof(sei), 0, NULL, NULL, szCmd, pszArgs };
```

Comment:
Wil this version of `ShellExecuteCommand` need a hWnd parameter one day? :thinking: (in comparison with the one in shlwapi)